app/components/add-validations-to-form.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if this.showRdfForm}}
+  <RdfForm
+    @groupClass="au-o-grid__item"
+    @form={{this.builderForm}}
+    @graphs={{this.graphs}}
+    @formStore={{this.builderStore}}
+    @forceShowErrors={{true}}
```

Comment:
Does this need to be set to `true`? This forces the error state for fields even if the user didn't touch them yet.